### PR TITLE
Remove dollar sign for command without output

### PR DESCRIPTION
**Pull request by kianmeng**
Easier to copy and paste command in GitHub and see https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md014.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,15 +239,49 @@ If you don't have 2FA enabled on PyPI already there's a section in the [PyPI Hel
...
+- Clone the GitHub repository locally. Don't use an existing clone of the repository to avoid tainting the workspace
+
+  ```sh
+  git clone ssh://git@github.com/sethmlarson/secure-python-package-template
+  ```
+
+- Check out the corresponding git tag.
+
+  ```sh
+  git checkout v0.1.0`)
```
Comment (sethmlarson) [inline README.md:251]: I believe the "`)" was unintentional here?

**Review by sethmlarson — COMMENTED** [review]
Love this, thank you so much! Totally agree that favoring copy-paste of users is the way to go. Only found one thing that I believe should be fixed:

**Review by sethmlarson — APPROVED** [review]
LGTM

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Comment by sethmlarson** [comment]
@kianmeng Thanks again for your contribution :)

**Pull request merged**